api/src/main/java/me/lokka30/treasury/api/economy/account/NonPlayerAccount.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/lokka30/Treasury>.
+ */
+
+package me.lokka30.treasury.api.economy.account;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.UUID;
+import me.lokka30.treasury.api.economy.response.EconomySubscriber;
+import me.lokka30.treasury.api.misc.TriState;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A Generic account is any non-player account.
```

Comment:
No longer called like that.
Since the NonPlayerAccount exists only to diferenciate player and non player accounts I think that should be mentioned on the description of the class